scissors/src/main/java/com/lyft/android/scissors/CropView.java

```diff
@@ -55,6 +57,10 @@
     private Matrix transform = new Matrix();
     private Extensions extensions;
 
+    private int shape = 0;
```

Comment:
I set up an IntDef (I find it more convenient when we need named constants) and added a comment pointing to the corresponding styleable attribute.